ports/unix/mpconfigport.h

```diff
@@ -90,6 +90,13 @@
 #define MICROPY_PY_ARRAY_SLICE_ASSIGN (1)
 #define MICROPY_PY_BUILTINS_SLICE_ATTRS (1)
 #define MICROPY_PY_SYS_EXIT         (1)
+#define MICROPY_PY_SYS_TRACE (0)
+#if MICROPY_PY_SYS_TRACE
+#define MICROPY_PY_SYS_UATEXIT (1)
+#define MICROPY_ACCESS_CODE_STATE (1)
```

Comment:
Why is `MICROPY_ACCESS_CODE_STATE` a separate config option?  It looks like it can't be enabled on its own, and `SYS_TRACE` requires it.

Reply:
Somehow I understood that it is desirable to have it as an independent feature #4342 .

Reply:
I see.  But it looks like the `current_code_state` variable is not restored on `FRAME_LEAVE` unless `SYS_TRACE` is also enabled, so this feature won't work by itself.  Anyway, I'd prefer to keep this PR as minimal as possible and not have extra configs like this.  They can be added later on if they prove useful.

Reply:
Awesome, I agree.